### PR TITLE
Document Admin Bot and Audit Bot

### PR DESCRIPTION
**Pull request by jaller94**
Fixes #68

### CODE REVIEW

**Comment by Twi1ightSparkle** [comment]
FYI I have a stale branch with some VIP docs you may be able to cherry pick from: https://github.com/vector-im/ems-docs/tree/twily/adminAuditBotDocs

**Comment by jaller94** [comment]
Thanks for pointing me to the old branch.
I've added the images and some other lines.

**Review by Twi1ightSparkle — CHANGES_REQUESTED** [review]
Both pages missing in the sidebar. Without being in SUMMARY.md they won't be compiled

File: src/integrations/Admin-Bot.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- Admin Bot is able to read encrypted messages to allow you to moderate messages.
+
+## Setup
+
+1. Go to the [Integrations tab on the EMS homeserver page](https://ems.element.io/user/hosting#/integrations).
+1. If you have more than one homesever, select the homeserver to add Admin Bot to.
+1. In the section Extensions, click on `Admin Bot`. If this is not visible, check that the homeserver is using the Element Enterprise Cloud plan.
+1. Click on `Set Up Integration` and confirm the pricing in a modal.
+
+![temp](/images/integrations/Admin-Bot/setup-button.png)
```
[the remark this answers is not in the document]
Comment (jaller94) [reply src/integrations/Admin-Bot.md:35]: I'd prefer to not have an alternative text as these screenshots don't show anything that hasn't been described as text.
I don't see a lot of value for visually impaired people and bots.

**Pull request merged**